packages/prisma/src/prisma/migrations/20250517215738_activity_log/migration.sql

```diff
@@ -0,0 +1,49 @@
+-- CreateEnum
+CREATE TYPE "ActivityLogType" AS ENUM ('MESSAGE', 'MODIFICATION', 'CREATION', 'SHARING');
+
+-- CreateTable
+CREATE TABLE "ActivityLogEntry" (
+    "id" SERIAL NOT NULL,
+    "type" "ActivityLogType" NOT NULL,
+    "objectType" "ObjectType" NOT NULL,
+    "message" TEXT,
+    "modificationDetails" JSONB,
+    "resolved" BOOLEAN NOT NULL DEFAULT false,
+    "resolvedAt" TIMESTAMP(3),
+    "userId" UUID,
+    "answerCollectionId" INTEGER,
+    "elementId" INTEGER,
+    "courseId" UUID,
+    "liveQuizId" UUID,
+    "practiceQuizId" UUID,
+    "microLearningId" UUID,
+    "groupActivityId" UUID,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL,
+
+    CONSTRAINT "ActivityLogEntry_pkey" PRIMARY KEY ("id")
+);
+
```

Comment:
TODO: add manual constraint to ensure one ID is set (end of PR when this does not have to be regenerated anymore)

Reply:
added object required constraint as present in similar tables